src/Writer/MySQL.php

```diff
@@ -152,25 +153,37 @@ public function write(CsvFile $csv, array $table): void
             ESCAPED BY ''
             IGNORE 1 LINES
             (". implode(', ', $columnNames) . ")
-            " . $this->emptyToNullOrDefault($table)
+            " . $this->getExpressionReplace($table)
         ;
 
         $this->logger->info(sprintf('Loading data to table "%s"', $table['dbName']));
-
         $this->exec($query);
     }
 
-    protected function emptyToNullOrDefault(array $table): string
+    protected function getExpressionReplace(array $table): string
     {
-        $columnsWithNullOrDefault = array_filter($table['items'], function ($column) {
-            return $this->hasColumnNullOrDefault($column);
-        });
+        $expressionsNullOrDefault = $this->emptyToNullOrDefault($table);
+        $expressionsReplaceValues = $this->replaceColumnsValues($table);
+
+        $expressions = array_merge(
+            iterator_to_array($expressionsNullOrDefault),
```

Comment:
@ondrajodas bude to fungovat aj pre nullable `BIT`? Dopln prosim test.

Reply:
@webrouse fixnuto a test je tu - https://github.com/keboola/db-writer-mysql/pull/40/commits/14b9635656bf6fd87f914fb27638404e7a02c5b4

Reply:
Ok. Ide to.

Bolo by lepsie ak by si mal `emptyToNullOrDefault` + `replaceColumnsValues` spojene do jednej metody, lebo prakticky robia to iste. Ak stlpec splna nejake podmienky, tak je potrebne nad nim vykonat nejaku operaciu.

Trocha sa bojim, ze sa tu bude nieco upravovat, a obidva metody vratia ten isty stlpec a bude duplicitne.